packages/autorest.python/test/azure/version-tolerant/specification/lropaging/README.md

```diff
@@ -0,0 +1,32 @@
+# Testing adding a custom poller and pager
+
+### Settings
+
+``` yaml
+title: LroPagingClient
+input-file: lropaging.json
+namespace: lropagingversiontolerant
+package-name: lropagingversiontolerant
+output-folder: $(python-sdks-folder)/azure/version-tolerant/Expected/AcceptanceTests/LroPagingVersionTolerant
+license-header: MICROSOFT_MIT_NO_VERSION
+clear-output-folder: true
+python: true
+package-version: 1.0.0b1
+basic-setup-py: true
+```
+
+```yaml
+directive:
+- where-operation: QuestionAnsweringProjects_UpdateQnas
```

Comment:
add test case with reference of real service: https://github.com/Azure/azure-sdk-for-python/blob/6f168b61101bbf851a30c085747d51470db4be0c/sdk/cognitivelanguage/azure-ai-language-questionanswering/swagger/README.md?plain=1#L114-L124